### PR TITLE
fix auth related bugs and calendar person found bug

### PR DESCRIPTION
**Pull request by DingmaomaoBJTU**
Problems:
1. Fix bot didn't response when user does not log in
2. Fix Calendar display user issue.

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Shared/ToDoSkillDialog.cs [hunk 1]
```diff
@@ -390,7 +390,15 @@ private Task<bool> TokenResponseValidator(PromptValidatorContext<Activity> pc, C
 
         private Task<bool> AuthPromptValidator(PromptValidatorContext<TokenResponse> pc, CancellationToken cancellationToken)
         {
-            return Task.FromResult(true);
+            var activity = pc.Recognized.Value;
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Shared/ToDoSkillDialog.cs:393]: Can you do this for ToDoSkill as well?

**Review by lzc850612 — CHANGES_REQUESTED** [review]
Please make the change for AuthPromptValidator for ToDoSkill as well

File: solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Shared/ToDoSkillDialog.cs (hunk 1 above)
Comment (DingmaomaoBJTU) [reply solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Shared/ToDoSkillDialog.cs:393]: Yes, I did this for email/calendar/todo~ This is already under todoskilldialog.cs (if I understand correctly?)

Comment (lzc850612) [reply solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Shared/ToDoSkillDialog.cs:393]: yeah this is good.

**Review by lzc850612 — APPROVED** [review]

**Review by youyu16 — APPROVED** [review]
I think is OK.

**Pull request merged**